admin/zabbix/Makefile

```diff
@@ -72,13 +74,21 @@ endef
 define Package/zabbix-server
   $(call Package/zabbix/Default)
   TITLE+= server
-  DEPENDS += +pgsql-cli +libevent2
+  DEPENDS += +mysql-server +libmariadb +libevent2
```

Comment:
this change seems completely out of scope of "add front end support"

Reply:
Done, now it makes complex of backend and database, where before it was only
CLI to connect exterior database. Now it looks like debian zabbix-server
package.